canary/gradle

```diff
@@ -1 +0,0 @@
-../gradle
```

Comment:
Why remove the symlink?

Reply:
On windows, gradlew.bat failed to download jar file into gradle folder because of it's a symlink.

We have to copy the whole gradle folder here.

Reply:
Doesn't this mean that this is broken across all of the djl-demo projects on windows?